controllers/goharbor/internal/test/certificate.go

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
IMO you should use fucntions from `controllers/goharbor/test_utils.go`, there is bugs here :-)